src/components/crossDevice/CrossDeviceSubmit/index.js

```diff
@@ -44,7 +44,7 @@ class CrossDeviceSubmit extends Component {
 
           <div>
             <button
-              className={`${theme.btn} ${theme["btn-primary"]} ${theme["btn-centered"]}`}
+              className={`${theme.btn} ${theme["btn-primary"]} ${theme["btn-centered"]} ${style.btn}`}
```

Comment:
not possible to change the style on the theme selectors?
If not than at least we should get rid of some selectors, 4 is quite a bit.